src/Munee/Asset/Type/Css.php

```diff
@@ -168,7 +169,8 @@ protected function isScss($file)
      */
     protected function fixRelativePaths($content, $originalFile)
     {
-        $cssParser = new CssParser($content);
+        $oSettings = CssSettings::create()->withMultibyteSupport(false);
```

Comment:
I did`t knew that.

This strings was taken directly from [Sabberworm readme](https://github.com/sabberworm/PHP-CSS-Parser). Next commit renames them as you want.